composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "~5.4"
+        "php": ">=5.4"
```

Comment:
who knows whether this works with 8 :)  Nobody could answer that question until 8 is at least underway.  Semver would say don't assume your code works past breaking versions (same reason we didn't let this say `>=5.4` already since that would've indicated that it worked for 7.0 when it did not).